modules/identity-provider-oidc-supported.adoc

```diff
@@ -5,7 +5,12 @@
 [id="identity-provider-oidc-supported_{context}"]
 = Supported OIDC providers
 
-The following OpenID Connect (OIDC) providers are tested and supported with {product-title}:
+Red Hat tests and supports certain OpenID Connect (OIDC) providers with {product-title}. The following list outlines these OIDC providers:
+
+[IMPORTANT]
+====
+If you use an OIDC provider with {product-title} that Red Hat did not test, consider that certain limitations might exist with using this provider with {product-title}. Regardless if this provider adheres to OpenID standards, limitations could still exist.
+====
```

Comment:
We can refer to https://developer.hashicorp.com/vault/docs/platform/k8s/helm 's statement which seems better:

_The following ... are currently supported ... It may work with other ..., but those are not supported._

```suggestion
The following OpenID Connect (OIDC) providers are tested and supported with {product-title}. It may work with other OIDC providers, but those are not supported.
```

Reply:
Thanks. Updated.